src/frontends/paddle/src/op/expand_v2.cpp

```diff
@@ -19,8 +19,16 @@ NamedOutputs expand_v2(const NodeContext& node) {
         auto inputs = node.get_ng_inputs("expand_shapes_tensor");
         ov::NodeVector node_vec;
         for (auto& input : inputs) {
+            if (input.get_partial_shape().rank().get_length() == 0) {
+                // should unsqueeze the input with non-shape.
+                auto unsqueeze_scalar = default_opset::Constant::create(ov::element::i32, {}, {0});
+                input = std::make_shared<default_opset::Unsqueeze>(input, unsqueeze_scalar);
+            }
+            PADDLE_OP_CHECK(node,
+                            input.get_partial_shape().rank().get_length() == 1,
+                            "the rank of conv input must == 1");
```

Comment:
Is this CHECK in need as it's after an unsqueeze node and should definitely be 1?

Reply:
> Is this CHECK in need as it's after an unsqueeze node and should definitely be 1?

I see. But PDPD export mechanism always changes from time to time. Sometimes it may be a 0-dim tensor. Sometimes it may be 2-dim tensor. So I have to leave an extra check here.